src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -614,6 +617,221 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
         );
     }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2, translate]);
 
+    const classifiedFields = [
```

Comment:
NAB: `classifiedFields` is a slightly confusing name. We could just call it `fields`.

Reply:
This is a binary classification

Reply:
Fair. Maybe we can add a comment explaining what we're doing though? Without that I don't know what it means for a field to be "classified". It only makes sense later on, which is fine, but not ideal.

Reply:
Suggested comment:

> An intermediate structure that helps us classify the fields as "primary" and "supplementary". The primary fields are always shown to the user, while an extra action is needed to reveal the supplementary ones

Reply:
@dukenv0307 @neil-marcellini What do you think about this comment suggestion?

Reply:
Yeah that's a great comment let's add it please, but also NAB